### ISSUE TITLE
Request for addition of mutation for relational operators in Gambit tool

### ISSUE THREAD

**Issue by Sta1400** [Certora/gambit#18]
As a user of the Gambit tool, I would like to suggest the addition of a mutation for relational operators, such as "<", ">", "<=", ">=", and "!=".

Currently, the Gambit tool offers a variety of mutations, including binary-operator-mutation and unary-operator-mutation, which can be used to mutate arithmetic and bitwise operators. However, there is no mutation available specifically for relational operators.

I believe that adding a mutation for relational operators would improve the effectiveness of mutation testing for Solidity code. Relational operators are frequently used in Solidity contracts to compare values, and introducing subtle changes to these operators can reveal hidden bugs and vulnerabilities in the code.

I understand that adding a new mutation can be time-consuming and requires careful consideration. However, I believe that this addition would greatly benefit the Solidity developer community and encourage more rigorous testing practices.

Thank you for your attention to this matter.

**Comment by BenTheKush** [comment]
I don't have this yet but it's going to happen soon!

**Comment by BenTheKush** [comment]
Just an update, these are in a PR that should be merged soonish...I'm stuck waiting on another PR in another tool to go through (it's taking forever...)
